frontend/src/visualizations/Scalar.css

```diff
@@ -0,0 +1,31 @@
+:local .Scalar {
+  display: flex;
```

Comment:
most of this declaration could be subsumed by using the atomic classes we have. really the only thing that seems necessary to specify here is the color which we might want to turn into a value in colors.css